### ISSUE TITLE
Warnings when using Unity Package Manager

### ISSUE THREAD

**Issue by mchamplain** [sschmid/Entitas-CSharp#708]
Hi,

I was testing the new Unity Package Manager (2018.1) by moving Entitas to a different folder and importing it in the manifest.json file.

Steps:
1) Import Entitas from Asset Store
2) Test some basic component + system (everything works fine)
3) move Entitas folder outside of the project
4) create package.json file in the new Entitas folder
5) update manifest.json file in Unity project to import custom Entitas package

manifest.json
```json
{
  "dependencies": {
    "com.unity.progrids": "3.0.2-preview",
    "com.desperatedevs.entitas": "file:../../../UnityCustomPackages/Entitas"
  }
}
```

package.json  (in UnityCustomPackages/Entitas)
```json
{
    "name": "com.desperatedevs.entitas",
    "displayName": "Custom Unity Package for Entitas",
    "version": "1.5.2",
    "unity": "2018.1",
    "description": "This package provides ...",
    "keywords": ["entitas"],
    "category": "Systems",
    "dependencies": {
    }
}
```

But using it that way generates a bunch of warnings in the console when running the game:
```
null texture passed to GUI.DrawTexture
UnityEngine.GUI:DrawTexture(Rect, Texture)
Entitas.VisualDebugging.Unity.Editor.EntitasHierarchyIcon:onHierarchyWindowItemOnGUI(Int32, Rect)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr)
```

**Comment by sschmid** [comment]
@mchamplain looks like the icons in the images folder are not found, or rather the meta files which contain the labels for those icons. VisualDebugging initially searches for those icons based on labels.

Are they missing?

**Comment by sschmid** [comment]
Any news?

**Comment by mchamplain** [comment]
@sschmid I didn't really investigate more at the time and moved Entitas back to Unity directly. But yesterday I tried to share the project with a colleague (via Git), and he got the exact same issue. When we deleted the Entitas folder from his side and re-downloaded the files from github, everything worked.

So I investigated a bit more and found that I was ignoring .meta files in my repo from the .gitignore in a parent folder. So yeah, the labels were gone when Unity regenerated the files.

Closed. Solution: make sure the .meta files are versioned controlled
